app/javascript/gobierto_data/webapp/components/commons/InfoBlockText.vue

```diff
@@ -47,6 +53,14 @@ export default {
       type: String,
       default: 'inherit'
     },
+    iconSize: {
+      type: String,
+      default: '16px'
+    },
+    fontSize: {
+      type: String,
+      default: '14px'
+    },
```

Comment:
What's the point of this, if you always call to this component with:

```vue
:icon-size="'10px'"
:font-size="'12px'"
```

Can't you spare them out?

Reply:
finally i use CSS instead props.